controlplane/kubeadm/controllers/scale_test.go

```diff
@@ -120,51 +120,6 @@ func TestKubeadmControlPlaneReconciler_scaleUpControlPlane(t *testing.T) {
 		g.Expect(fakeClient.List(ctx, &controlPlaneMachines)).To(Succeed())
 		g.Expect(controlPlaneMachines.Items).To(HaveLen(3))
 	})
-	t.Run("does not create a control plane Machine if preflight checks fail", func(t *testing.T) {
```

Comment:
Why was this removed?

Reply:
@vincepri Ups, my mistake. I reversed these change.